src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2_eink.py

```diff
@@ -221,9 +227,11 @@ def arbeitsl_geld_2_eink_anr_frei_m(
     # Beneficiaries who live with a minor child in a group home or who have a minor
     # child have slightly different thresholds. We currently do not consider the second
     # condition.
+    eink_summe = bruttolohn_m + eink_selbst_m
```

Comment:
```suggestion
    eink_erwerbstätigkeit = bruttolohn_m + eink_selbst_m
```

I think the technical term is "Einkommen aus Erwerbstätigkeit".